### ISSUE TITLE
unistd.h problem

### ISSUE THREAD

**Issue by Kabron287** [takkaO/M5_LCD_Emulator#8]
I could not realize Sleep macro under Windows in any way.
During standard compilation VS does not found unistd.h
I tried to use this one:
**https://github.com/robinrowe/libunistd**
but got a lot of errors.

What is the solution?
Thanks in advance.

**Comment by takkaO** [comment]
Hi Kabron287
**Thank you for using this programs!**

By Sleep macro do you mean Sleep() in the delay function (in M5Stack.cpp/.h)?
If you are using Windows OS, Sleep() is called from ```windows.h```. 
Therefore, ```unistd.h``` is not required in Windows OS. (```ifndef LINUX``` in ```M5Stack.h``` to distinguish between the Windows and Linux.)

For more detailed advice, more information is needed.
For example,
- What version of Visual Studio you're using.
- The project's include settings.
- What OS you're using (7 or 8 or 10?).
- A capture of the error screen.
- Zip file of the your entire program code ...etc

Thank you.

**Comment by Kabron287** [comment]
Hello,
thanks for answer.
1. VS 2017
2. All as described
3. Windows 10 x64 LTSC
4. unistd.h not found
5. Attached without .vs else too big for upload
[M5_LCD_Emulator-develop.zip](https://github.com/takkaO/M5_LCD_Emulator/files/5395936/M5_LCD_Emulator-develop.zip)


It is the only problem. If I do not use Sleep everything works OK, but with zero delay.

**Comment by takkaO** [comment]
I was able to work your project.
First of all, the ```unistd.h not found``` error did not appear in my environment.
So, perhaps the path to ```libunistd``` is wrong, or the project structure is different (```Release``` or ```Debug```).

Instead, I get the error ```C2872: ACCESS_MASK: ambiguous symbol. ```
This is due to a name conflict between ```unistd.h``` and opencv.
The solution is to remove ```using namespace cv``` from all files and to decorate some method  (For example, ```Mat``` to ```cv::Mat```).　This is so annoying...


After above, I passed build project.
However, ```usleep()``` did not work properly.
The ```usleep()``` argument predict microseconds, but the behavior is nanoseconds.
Although, it seems to be working correctly (if we use nanoseconds).

I upload my work project.
[M5_LCD_Emulator-develop.zip](https://github.com/takkaO/M5_LCD_Emulator/files/5397536/M5_LCD_Emulator-develop.zip)

I edited the following files.
- main.hpp
- M5Stack.cpp
- M5Stack.h
- M5Emulator.cpp
- M5Emulator.hpp

Sorry for writing such a long message.
